installer.sh

```diff
@@ -525,11 +518,11 @@ uninstall (){
   # delete user www-embark and reset visudo
   echo -e "$ORANGE""$BOLD""Delete user""$NC"
   # sed -i 's/www\-embark\ ALL\=\(ALL\)\ NOPASSWD\:\ \/app\/emba\/emba.sh//g' /etc/sudoers #TODO doesnt work yet
-  userdel www-embark
+  userdel www-embark || true
 
   # delete .env
   echo -e "$ORANGE""$BOLD""Delete env""$NC"
-  rm ./.env
+  rm ./.env || true
```

Comment:
please always check if the file is available before doing something with it:
if [[ -f ./.env ]]; then
rm
fi